batch/batch/front_end/front_end.py

```diff
@@ -203,22 +209,22 @@ async def rest_get_supported_regions(request, userdata):  # pylint: disable=unus
     return json_response(list(request.app['regions'].keys()))
 
 
-async def _handle_ui_error(session, f, *args, **kwargs):
+async def _handle_ui_error(session, f, *args, **kwargs) -> Tuple[bool, Any]:
     try:
-        await f(*args, **kwargs)
+        result = await f(*args, **kwargs)
     except KeyError as e:
         set_message(session, str(e), 'error')
         log.info(f'ui error: KeyError {e}')
-        return True
+        return (True, None)
     except BatchOperationAlreadyCompletedError as e:
         set_message(session, e.message, e.ui_error_type)
         log.info(f'ui error: BatchOperationAlreadyCompletedError {e.message}')
-        return True
+        return (True, None)
     except BatchUserError as e:
         set_message(session, e.message, e.ui_error_type)
         log.info(f'ui error: BatchUserError {e.message}')
-        return True
-    return False
+        return (True, None)
+    return (False, result)
```

Comment:
This introduces another style of error handling into the codebase that I think harms consistency. While very idiomatic [go](https://go.dev/blog/error-handling-and-go), I think it would be better to stick with the pythonic style of error-handling where this function either returns the return value of `f` or raises an exception if `f` fails. Then callers should use this in a try-except block instead of using `if errored`.